husky_bringup/launch/dual_ur5_arm_config/right_ur5.launch

```diff
@@ -0,0 +1,69 @@
+   following disclaimer in the documentation and/or other materials provided with the distribution.
+ * Neither the name of Clearpath Robotics nor the names of its contributors may be used to endorse or promote
+   products derived from this software without specific prior written permission.
+
+THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS OR IMPLIED WAR-
+RANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, IN-
+DIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT
+OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+-->
+
+<launch>
+ 
+   <!-- THIS IS FOR THE RIGHT ARM ON THE DUAL UR5 HUSKY -->
+
+  <remap from="/husky_right_ur5/joint_states" to="/joint_states"/>
+
+ <arg name="dual_ur5_enabled" default="$(optenv HUSKY_DUAL_UR5_ENABLED false)" />
+  <group if="$(optenv HUSKY_DUAL_UR5_ENABLED false)" ns="husky_right_ur5">
+    <!-- Launches the drivers and sets parameters for the ur5 arm -->
+    <!-- Mostly taken from the ur_common.launch inside ur_common.launch -->
+
+    <arg name="robot_ip" default="$(env HUSKY_RIGHT_UR5_IP)"/>
+    <arg name="reverse_port" default="50002"/>
+    <arg name="min_payload" default="0.0"/>
+    <arg name="max_payload" default="10.0"/>
+    <arg name="prefix" default="r_ur5_arm_"/>
+
+    <!-- The max_velocity parameter is only used for debugging in the ur_driver. It's not related to actual velocity limits -->
+    <arg name="max_velocity" default="10.0"/> <!-- [rad/s] -->
+
+    <!-- copy the specified IP address to be consistant with ROS-Industrial spec.
+    NOTE: The ip address is actually passed to the driver on the command line -->
+    <param name="robot_ip_address" type="str" value="$(arg robot_ip)"/>
+    <param name="robot_reverse_port" type="int" value="$(arg reverse_port)"/>
+
+    <!-- driver -->
+    <node name="right_ur5_driver" pkg="ur_modern_driver" type="ur_driver" args="$(arg robot_ip) $(arg reverse_port)" output="screen" respawn="true">
+      <param name="min_payload" type="double" value="$(arg min_payload)"/>
+      <param name="max_payload" type="double" value="$(arg max_payload)"/>
+      <param name="max_velocity" type="double" value="$(arg max_velocity)"/>
+      <param name="prefix" type="string" value="$(arg prefix)"/>
+      <param name="reverse_port" value="$(arg reverse_port)"/>
+    </node>
+
+    <!-- TF Buffer Server -->
+    <node pkg="tf2_ros" type="buffer_server" name="right_buffer_server" respawn="true">
```

Comment:
? this gets installed using husky_bringup install and it is used to launch the UR5 arms for the husky. However we did determine that the user should launch these manually and so these should be moved somewhere else... But not sure where.... the existing scripts aren't namespaced, and don't use env vars.